pkg/assembler/backends/ent/backend/sbom.go

```diff
@@ -80,13 +81,16 @@ func (b *EntBackend) IngestHasSbom(ctx context.Context, subject model.PackageOrA
 			SetDigest(strings.ToLower(spec.Digest)).
 			SetDownloadLocation(spec.DownloadLocation).
 			SetOrigin(spec.Origin).
-			SetCollector(spec.Collector)
+			SetCollector(spec.Collector).
+			SetKnownSince(spec.KnownSince)
```

Comment:
this should be UTC like how you did the arango changes

Reply:
Changed, thanks!